File: C++/2469.温度转换.cpp
/*
 * @lc app=leetcode.cn id=2469 lang=cpp
 *
 * [2469] 温度转换
 */

// @lc code=start
class Solution
{
public:
    vector<double> convertTemperature(double celsius)
    {
        return vector<double>{(celsius * 100 + 27315) / 100, (celsius * 180 + 3200) / 100};
    }
};
// @lc code=end

File: JavaScript/2469.温度转换.js
/*
 * @lc app=leetcode.cn id=2469 lang=javascript
 *
 * [2469] 温度转换
 */

// @lc code=start
/**
 * @param {number} celsius
 * @return {number[]}
 */
var convertTemperature = function (celsius) {
  return [celsius + 273.15, celsius * 1.8 + 32.0];
};
// @lc code=end
